test/breadcrumbs-test.js

```diff
@@ -192,6 +192,30 @@ describe('Breadcrumbs', () => {
             expect(res[2].type.displayName).to.equal('Link');
         });
 
+        it('should ignore routes without path', () => {
```

Comment:
Consider adding a negative test as well, e.g how it is defined if `hideNoPath` is `false`. 
All other tests have "well-defined" routes so this is covered.